standalone/src/main/resources/cz/crcs/ectester/standalone/libs/jni/Makefile

```diff
@@ -96,14 +96,11 @@ openssl.o: openssl.c
 # BoringSSL shim
 boringssl: boringssl_provider.so
 
-lib_boringssl.so:
-	cp $(PROJECT_ROOT_PATH)/ext/boringssl/build/crypto/libcrypto.so lib_boringssl.so
-
-boringssl_provider.so: boringssl.o c_utils.o | lib_timing.so lib_csignals.so lib_boringssl.so
-	$(CC) $(LFLAGS) -o $@ -Wl,-rpath,'$$ORIGIN/lib' $^ -L. lib_boringssl.so -l:lib_timing.so -l:lib_csignals.so
+boringssl_provider.so: boringssl.o c_utils.o | lib_timing.so lib_csignals.so
+	NIX_CFLAGS_COMPILE= $(CC) $(CFLAGS) $(LFLAGS) -o $@ -Wl,-rpath,'$$ORIGIN/lib' $^ -L. -Wl,-Bstatic -l:lib_boringssl.a -Wl,-Bdynamic -l:lib_timing.so -l:lib_csignals.so
 
 boringssl.o: boringssl.c
-	$(CC) -I$(PROJECT_ROOT_PATH)/ext/boringssl/include/ $(CFLAGS) -c $<
+	NIX_CFLAGS_COMPILE= $(CC) -I$(BORINGSSL_CFLAGS) $(CFLAGS) -c $<
```

Comment:
Ideally, this Makefile will still remain useful to a user that does not use nix. Can we keep that? Or back it up somewhere? Essentially I would like to keep a simple Makefile somewhere for someone that has the libraries installed system-wide (or locally) and just wants to build them.

The use case is also testing custom versions of the libraries and not having to fuff with nix.

Reply:
By custom I mean when someone is actively working on them/patching them.